includes/constants.php

```diff
@@ -3,13 +3,22 @@
 namespace Automattic\WP\Cron_Control;
 
 /**
- * Job queues
+ * Adjustable queue size and concurrency limits, to facilitate scaling
+ */
+$job_queue_size = defined( 'CRON_CONTROL_JOB_QUEUE_SIZE' ) && is_numeric( \CRON_CONTROL_JOB_QUEUE_SIZE ) ? absint( \CRON_CONTROL_JOB_QUEUE_SIZE ) : 10;
+define( __NAMESPACE__ . '\JOB_QUEUE_SIZE', $job_queue_size );
+unset( $job_queue_size );
+
+$job_concurrency_limit = defined( 'CRON_CONTROL_JOB_CONCURRENCY_LIMIT' ) && is_numeric( \CRON_CONTROL_JOB_CONCURRENCY_LIMIT ) ? absint( \CRON_CONTROL_JOB_CONCURRENCY_LIMIT ) : 10;
```

Comment:
Minor: for readability I would avoid the ternary here and go for an explicit `if`:

```
$job_concurrency_limit = 10;
if ( defined( 'CRON_CONTROL_JOB_CONCURRENCY_LIMIT' ) && is_numeric( \CRON_CONTROL_JOB_CONCURRENCY_LIMIT ) ) {
    $job_concurrency_limit = \CRON_CONTROL_JOB_CONCURRENCY_LIMIT; // max/min from other comment would go here
}
...
```